Package.swift

```diff
@@ -29,7 +29,7 @@ let package = Package(
             name: "Auth0", 
             dependencies: ["SimpleKeychain", "JWTDecode", "Auth0ObjectiveC"], 
             path: "Auth0",
-            exclude: ["ObjectiveC"],
+            exclude: ["ObjectiveC", "Info.plist", "Info-tvOS.plist"],
```

Comment:
Swift tools version 5.3+ require declaring (bundling) any resource files. If there are non-code files that are not being bundled, it will produce a warning. So we need to exclude those files.
See https://developer.apple.com/documentation/swift_packages/bundling_resources_with_a_swift_package